sycl/doc/developer/WorkingOnAReleaseBranch.md

```diff
@@ -0,0 +1,39 @@
+# Working on a release branch
+
+Release branch is defined as a branch whose name starts with `sycl-rel-` prefix.
+
+Those branches are intended to indicate stable snapshots of our product so that
+our users don't need to guess which nightly build is good enough for our needs.
+
+Therefore, those branches have higher quality requirements and as such have
+different contribution rules intended to preserve their stability.
+
+If you are not familiar with [general contribution guidelines][contributing] or
+[DPC++ specific contribution guidelines][contributing-to-dpcpp], please
```

Comment:
```suggestion
If you are not familiar with the [general contribution guidelines][contributing] or
the [DPC++ specific contribution guidelines][contributing-to-dpcpp], please
```

Reply:
Applied in 46c917a90f8b3e25e1a8e30e1d410db61ab323e3